0008-security-recommendations/README.md

```diff
@@ -0,0 +1,98 @@
+---
+name: KP Security Recommendations
+shortname: 8/Security
+status: raw
+editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
+---
+## Introduction
+This document describes security recommendations for systems using 1/KP. This is only a recommendation and is not mandatory to implement.
+
+## Model
+We start with the generalization of the system to better understand the attack surface, threats, and possible protection techniques.
+
+The general system consists of a server, clients, and end devices. End devices are devices that produce data, and the client only serves as a proxy for them. End devices are represented as endpoints within 1/KP.
+
+So the connection scheme looks like this:
+```
+server -- clients -- end devices
+```
+
+All data flows between the server and end devices via clients.
+
+As you see, there are two links involved: server-client and client-device. These links are purely logical. (e.g., there is no physical link if the client acts as an end device and produces data on its own.)
```

Comment:
True. My point is: there is still an option that the client is not trusted (enough) to handle the endpoint data, so it appears that in addition to the two links depicted, there might be an EP-server link?